templates/articles/_comment.html

```diff
@@ -0,0 +1,27 @@
+<turbo-frame id="comment-{{ comment.pk }}">
+<div class="card">
+    <div class="card-block">
+        <p class="card-text">
+            {{ comment.body }}
+        </p>
+    </div>
+    <div class="card-footer">
+        <a href="" class="comment-author">
+            <img
+                    src="{{ comment.author.img }}"
+                    class="comment-author-img"
+            />
+        </a>
+        &nbsp;
+        <a href="" class="comment-author">{{ comment.author }}</a>
+        <span class="date-posted">{{ comment.created_at }}</span>
+
+        {% if user.pk == comment.author.pk %}
+        <span class="mod-options">
+              <i class="ion-edit"></i>
+            <a href="{% url 'delete_comment' comment.pk %}"><i class="ion-trash-a"></i></a>
+            </span>
+        {% endif %}
+    </div>
+</div>
+</turbo-frame>
```

Comment:
Comments could be a fun spot to use Turbo Streams to get live updates! Not in this PR though :)